data/json/items/armor/suits_protection.json

```
@@ -1400,7 +1400,7 @@
         "coverage": 85,
         "material": [
           { "type": "plastic_pad", "covered_by_mat": 100, "thickness": 6 },
-          { "type": "thermo_resin", "covered_by_mat": 85, "thickness": 1.5 }
+          { "type": "thermo_resin", "covered_by_mat": 80, "thickness": 1.5 }
```

```suggestion
          { "type": "thermo_resin", "covered_by_mat": 70, "thickness": 1.5 }
```
The riot armour I see still leaves a lot of open space on the underarms and places like that, the hard plates only cover the outside of the arm. Those would definitely absorb a majority of attacks but I don't think we should oversell them.